packages/world-modules/mud.config.ts

```diff
@@ -190,7 +190,7 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
-    TokenURI: {
+    ERC721TokenURI: {
```

Comment:
Maybe ERC721 and ERC1155 could share tables like `TokenURI` or `OperatorApproval` that have the same schema? 

Just like `Balances`, in the `modules/tokens/tables` directory: https://github.com/MerkleBoy/mud/blob/merkleboy/erc1155-puppet-module/packages/world-modules/mud.config.ts#L121

Reply:
Thanks ! It's a really rough draft but I thought it'd be wise to make a draft now (even if it doesn't quite compile haha)

I wasn't sure about the data structures, but you're completely right that I should merge the tables that are constructed the same way. I'll include those changes in the next commit